stream_chat/async_chat/channel.py

```diff
@@ -139,13 +139,14 @@ async def delete(self):
         """
         return await self.client.delete(self.url)
 
-    async def truncate(self):
+    async def truncate(self, **options):
         """
         Removes all messages from the channel
 
+        :param options: the query options, check docs on https://getstream.io/chat/docs/
```

Comment:
```suggestion
        :param options: the query options, check docs on https://getstream.io/chat/docs/python/channel_delete/?language=python#truncating-a-channel
```

could you also add python code snippet to the docs?

Reply:
yea sounds good, better to do it in another PR

Reply:
Pushed the fix. I'll try to update the docs as well.

The tests are failing because there's a bug in the backend. @ferhatelmas opened a PR to fix it.